iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/NetworkImpl.java

```diff
@@ -966,6 +966,11 @@ private void mergeDanglingLines(List<MergedLine> lines, DanglingLine dl1, Dangli
         if (dl1 != null) {
             MergedLine l = new MergedLine();
             l.id = dl1.getId().compareTo(dl2.getId()) < 0 ? dl1.getId() + " + " + dl2.getId() : dl2.getId() + " + " + dl1.getId();
+            l.aliases = new HashSet<>();
+            l.aliases.add(dl1.getId());
+            l.aliases.add(dl2.getId());
+            l.aliases.addAll(dl1.getAliases());
+            l.aliases.addAll(dl2.getAliases());
```

Comment:
Should we change the mergeability check to consider also aliases?

Reply:
Should we enforce consistency (no duplicate aliases) or remove them in case of conflict during merging ?

Reply:
What is the current behavior if you try to add an existing alias to another identifiable? If I remember well, an exception is thrown. So I think we should consider the network are not mergeable, specially if we consider alias as a kind of identifier.